ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -214,6 +214,15 @@ def run(self):
         cli_data['bgp_v6'] = self.check_series_status(data, "bgp_route_v6", "BGP v6 routes")
         cli_data['po']     = self.check_change_time(samples, "po_changetime", "PortChannel interface")
 
+        route_timeout             = log_data['route_timeout']
+        cli_data['route_timeout'] = route_timeout
+
+        # {'10.0.0.38': [(0, '4200065100)')], 'fc00::2d': [(0, '4200065100)')]}
+        for nei in route_timeout.keys():
+            asn = route_timeout[nei][0]
```

Comment:
is it 0 as asn in your example above?

Reply:
My ans is 4200065100

Reply:
Here is the error message for timeout:

FAILED:10.64.247.203:BGP route GR timeout: neighbor fc00::2d ASN (0, '4200065100)')

So I guess "asn = route_timeout[nei][0][-1]" works better?

Reply:
Too long ASN. Are you sure this number is you ASN?

Reply:
The new  error message: "FAILED:10.64.247.203:BGP route GR timeout: neighbor 10.0.0.38 (ASN 4200065100)"

Yes, I am testing 4 bytes ASN on 7260cx3s.

Reply:
route_timeout[nei][0] will extract first element of a tuple. You need a second one if I understand you well